### ISSUE TITLE
Dogstatsd's ServiceCheck + Metrics' HealthCheck

### ISSUE THREAD

**Issue by edudar** [coursera/metrics-datadog#51]
`metrics-healthchecks` module provides a way to check and expose status of external systems that particular service depends on. Dogstatsd has very similar concept of `ServiceCheck`. That would be useful to tie them together so that applications can reports service checks for screen type of dashboards in DataDog.

I'll see if I have time to work on PR for this feature.

**Comment by nickdella** [comment]
Good idea, @edudar !  This sounds like a clean bridge between Dropwizard and Datadog functionality.  Your PR would be most welcome :)

**Comment by nishantrayan** [comment]
@edudar do you know if there is any ongoing work to do this. We need the same functionality and happy to contribute

**Comment by edudar** [comment]
@nishantrayan Feel free to take the lead

**Comment by edudar** [comment]
Note, however, is that service checks are [deprecated](https://docs.datadoghq.com/integrations/faq/using-events-for-service-checks-is-deprecated-in-favor-of-monitors/) by Datadog as of now and replaced with monitors and HTTP check in particular so it might not be worth it.
